x-pack/plugins/synthetics/public/apps/synthetics/lib/alert_types/common.ts

```diff
@@ -0,0 +1,10 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+export const getUrlForAlert = (id: string, basePath: string) => {
```

Comment:
We should avoid using a hardcoded link like this, and instead use locators. Management already has some defined at `src/plugins/management/common/locator.ts`